sklearn/preprocessing/imputation.py

```diff
@@ -177,6 +178,10 @@ def fit(self, X, y=None):
                                                    self.strategy,
                                                    self.missing_values,
                                                    self.axis)
+            invalid_mask = np.isnan(self.statistics_)
+            self.all_missing_ = np.where(invalid_mask)[0]
```

Comment:
IMHO I don't know if this attribute would be of any great help but it needs it to be stored since the  transform function should know the features so that it can raise a ``RuntimeWarning`` in case that feature has some non-missing values in ``transform``.